server-spi/src/main/java/org/keycloak/models/credential/dto/RecoveryAuthnCodesCredentialData.java

```diff
@@ -0,0 +1,48 @@
+package org.keycloak.models.credential.dto;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class RecoveryAuthnCodesCredentialData {
+
+    private final int hashIterations;
+    private final String algorithm;
+
+    private int totalCodes;
+    private int remainingCodes;
+
+    @JsonCreator
+    public RecoveryAuthnCodesCredentialData(@JsonProperty("hashIterations") int hashIterations,
+            @JsonProperty("algorithm") String algorithm, @JsonProperty("remaining") int remainingCodes,
+                                            @JsonProperty("total") int totalCodes) {
+        this.hashIterations = hashIterations;
+        this.algorithm = algorithm;
+        this.remainingCodes = remainingCodes;
```

Comment:
It looks there is missing line:
```
this.totalCodes = totalCodes;
```
This causes that admin console always shows 0 total codes, which is incorrect.